src/main/java/at/hannibal2/skyhanni/data/mob/MobFilter.kt

```diff
@@ -55,7 +55,7 @@ object MobFilter {
     private val patternGroup = RepoPattern.group("mob.detection")
 
     @Language("RegExp")
-    private val mobType = "(?<mobType>[^\\s✯]+ )?"
+    private val mobType = "(?<mobType>[^\\w\\s✯-]+ )?"
```

Comment:
Actually - should be escaped. Or else you get unintended behavior. But idk why it even is in the list.
```suggestion
    private val mobType = "(?<mobType>[^\\w\\s✯\-]+ )?"
```

Reply:
It shouldn't be necessary to escape if it's at the beginning or end of a pattern, that's a special case.

I added it just in case because I could see it being part of a mob name, but it's probably technically not necessary because the word would have to also not contain any alphanumeric characters for it to match.

Reply:
Yeah and if someone who doesn't know about that fact adds a character to the end, unintended behavior will occur.

Reply:
Fair enough.